org.eclipse.m2e.pde.target/src/org/eclipse/m2e/pde/target/MavenTargetBundle.java

```diff
@@ -139,11 +140,9 @@ public static TargetBundle getWrappedArtifact(Artifact artifact, Properties bndI
 					wrappedFile.setLastModified(artifactFile.lastModified());
 				}
 			}
-			TargetBundle targetBundle = new TargetBundle(wrappedFile);
 			try (FileOutputStream os = new FileOutputStream(instructionsFile)) {
 				bndInstructions.storeToXML(os, null);
 			}
-			return targetBundle;
 		}
 		try {
```

Comment:
You're talking about line 147, right? Because I can't see anything out of the ordinary.
The code is also consistently using tabs, so that isn't it, either. 
![image](https://user-images.githubusercontent.com/92990747/172395696-ac82e2bb-92de-4d62-a21b-e4a446be0d26.png)

Reply:
Strange... might be github UI then....